misc/to_be_appended_into_innovation.css

```diff
@@ -0,0 +1,962 @@
+/* Auto-generated by css_generator.py (Python script) */
```

Comment:
Probably could add a gitignore here, but it could be of worth for peer reviewing changes that are brough upon the generator.

Reply:
Yeah it seems fine to check this in. Especially since it's in misc/ now.